package.json

```diff
@@ -26,9 +26,9 @@
   "scripts": {
     "lint": "npm-run-all --continue-on-error --parallel lint:*",
     "lint:docs": "markdownlint \"**/*.md\"",
-    "lint:docs:fix": "yarn lint:docs --fix",
+    "lint:docs:fix": "npm run lint:docs --fix",
```

Comment:
To pass options to these NPM commands, I believe we need `-- --fix` for this to work...

Reply:
Another good catch.

Reply:
Fixed.